### DISCUSSION TITLE
MockData (json파일) 관련

### DISCUSSION THREAD

**Discussion by baechanghyeon** [WantedPreonboardingFE5team/pre-onboarding-7th-2-1-5#26] [General]
- MockData를 어느 폴더에서 관리하는게 맞을까요?
 이번 과제에서는 API가 없으니까 apis폴더 여기에 넣는게 맞을까요? 아니면 utils폴더에다가 넣어놔야 할까요?

**Comment by yeondooo** [comment]
저는 db폴더를 만들어서 관리했습니다!
<img width="159" alt="image" src="https://user-images.githubusercontent.com/97602748/200112686-7ff6cc51-c096-4669-9085-7eb7cb1b2d85.png">

**Reply by baechanghyeon** [reply]
오 db 파일명도 괜찮아보이네요! ㅎㅎ 일단 저도 이렇게 넣고 작업하겠슴니다

**Comment by seungyeon-rr** [comment]
![image](https://user-images.githubusercontent.com/106301980/200119801-dc2e0cc3-320f-405f-92fa-f7e5c6f05886.png)
전 assets 폴더에 data 폴더 만들어서 넣었습니다! (이렇게 배워서...ㅎㅎ)
